cypress/e2e/pages/photoGalleryPage/index.cy.js

```diff
@@ -112,7 +102,7 @@ const liteTestSuites = canonicalTestSuites
     return {
       ...testSuite,
       path: `${testSuite.path}.lite`,
-      tests,
+      tests: [testsForAllPages],
```

Comment:
Can we also import the lite tests from: https://github.com/bbc/simorgh/blob/main/cypress/e2e/pages/articles/testsForLiteOnly.js - I thought we had separated them out so that we could reuse for STY pages? If not, it's OK to import from the article page for now